images/status_indicator.go

```diff
@@ -0,0 +1,56 @@
+package images
+
+import (
+	"bytes"
+	"image"
+	"image/color"
+	"image/png"
+	"math"
+
+	"github.com/fogleman/gg"
+)
+
+func AddStatusIndicatorToImage(inputImage []byte, innerColor color.Color, indicatorSize, indicatorBorder int) ([]byte, error) {
```

Comment:
so even an indicator in status-go? how it will be updated? seems like its better to have it in react no?

Reply:
> seems like its better to have it in react no?

That's what I planned for in the first place.
Tried my best to remember why I chose to do that in status-go.
 Found out, in the figma design, the indicator border shold be transparent so that the color of it shold be whatever beneath it. Although, I just checked, it's not the case in old user-avatar component